src/Molecules/ProfileCard/ProfileCardStyled.js

```diff
@@ -3,7 +3,6 @@ import styled from 'styled-components';
 const ProfileCardStyled = styled.div`
   display: flex;
   flex-direction: column;
```

Comment:
maybe you can delete this file and use ColumnWrapper?
![image](https://user-images.githubusercontent.com/75560322/152025589-0f7a16b8-d3f8-48ce-801e-6f8a80e138b2.png)